{{cookiecutter.directory_name}}/pyproject.toml

```diff
@@ -10,6 +72,18 @@ branch = true
 source = ["{{ cookiecutter.package_name }}"]
 command_line = "-m pytest"
 
+[tool.isort]
+lines_after_imports = 2
+force_single_line = 1
+no_lines_before = ["FUTURE", "STDLIB", "THIRDPARTY", "FIRSTPARTY", "LOCALFOLDER"]
+known_first_party = "{{ cookiecutter.package_name }}"
+src_paths = ["{{ cookiecutter.package_name }}", "tests"]
+line_length = 120
+
+[tool.setuptools]
+include-package-data = true
+packages = ["{{ cookiecutter.package_name }}"]
+
```

Comment:
```suggestion

[tool.coverage.run]
branch = true
source = ["{{ cookiecutter.package_name }}"]


[tool.setuptools.packages.find]
include = ["{{ cookiecutter.package_name }}*"]
exclude = ["tests*"]
```

Reply:
I think if we would move to a `src`-based layout (see my review comment) of the part above, we can also get rid of the `tool.setuptools.packages.find` section you suggest. Then the `source` property can be simply `source = ["src"]`.